reference/orders.v2.oas2.yml

```diff
@@ -3082,20 +3082,19 @@ components:
               tax_provider_id:
                 type: string
                 description: |
-                  BasicTaxProvider - Tax is set to manual.
+                  BasicTaxProvider - Tax is set to manual and order is created in the store.
 
-                  AvaTaxProvider - This is for when the tax provider has been set to automatic and the order was NOT created by the API. Used for Avalara.
+                  AvaTaxProvider - Tax is set to automatic and order is created in the store. Used for Avalara.
```

Comment:
Seems like the tax provider is Avalara and the tax provider was automatically set on the order (probably by the Avalara integration). Would another integration literally use this string, get a null because it used the API, or would it get its own new string?. It's an awkward set of things to say together, but IMO worth digging because of all the new tax provider services, like TaxJar.

Reply:
@WillemHoman does this statement --what Sarah said-- still stand here? Would another integration use the string `AvaTaxProvider`?

Reply:
This will be tax provider specific. So Avalara will be `AvaTaxProvider`, Basic Tax is `BasicTaxProvider`. 
This list will expand over time.
I just [pinged the tax team](https://bigcommerce.slack.com/archives/C97B8DEGK/p1646864901260169) on this and there is no publicly available list of tax providers and tax provider ids. 
Apparently it also varies by region, so this sounds like something that should be documented for tax and linked to from  the orders api docs?
Would a couple of examples suffice for now?

Reply:
That would be great to add for clarity. Thanks so much @WillemHoman.